packages/core-bridge/src/conversions.rs

```diff
@@ -49,6 +53,17 @@ impl ArrayHandleConversionsExt for Handle<'_, JsArray> {
         }
         Ok(ret_vec)
     }
+
+    fn to_vec_of_float(&self, cx: &mut FunctionContext) -> NeonResult<Vec<f64>> {
```

Comment:
This could be a generic that takes <JSxxx, RustType> if desired

Reply:
Yeah, I'm definitely planing more refactoring of the type helpers and macros.